package.json

```
@@ -173,6 +175,7 @@
     "check-image-imports": "node scripts/checkForOrphanImports.js",
     "clean": "gatsby clean",
     "convert-to-webp": "node scripts/convertPNGs.mjs",
+    "create-docs-and-last-modified-csv": "./scripts/docsAndLastDateEdited.mjs",
```

this is a loong boi. what about something like:
```suggestion
    "freshness": "./scripts/docsAndLastDateEdited.mjs",
```
?